frontend/src/components/display-products/ProductQuantity.tsx

```diff
@@ -1,33 +1,27 @@
-import React from "react";
+import { FC } from "react";
 
 import { Typography } from "@mui/material";
 import useShoppingCartContext from "../../hooks/context/useShoppingCartContext";
 import { Product } from "../../contexts/shopping-cart-context/Types";
 
-interface ProductQuantityProps {
-    name: string | undefined;
-}
+import { quantityStyle } from "./styles";
+import { ProductQuantityProps } from "./Types";
+import { QUANTITY_TEXT } from "../constants/text";
 
-const ProductQuantity: React.FC<ProductQuantityProps> = ({ name }) => {
-    const { shoppingCartItems } = useShoppingCartContext()
+const ProductQuantity: FC<ProductQuantityProps> = ({ name }) => {
 
-    const quantityStyle = {
-        marginTop: "7px",
-        fontWeight: "bold",
-    };
+    const { shoppingCartItems } = useShoppingCartContext()
 
     return (
-        <div id="quantity">
-            { shoppingCartItems.map((item: Product) => {
-                const { productId, quantity } = item || {};
-                if (item?.productId === name) {
-                    return (
-                        <Typography sx={ quantityStyle } key={ productId }>
-                            { quantity } of this product in cart
-                        </Typography>
-                    );
-                }
-                return null;
+        <div id="product-quantity">
+            { shoppingCartItems?.map((item: Product) => {
+                const { productId, quantity } = item
+                const isMatchingProductId = productId === name;
```

Comment:
name of what?
it feels weird that name should match with and id, could you please fix this?